packages/wonder-blocks-link/src/components/link-core.tsx

```diff
@@ -165,7 +152,8 @@ const linkContentStyles = StyleSheet.create({
         marginInlineStart: Spacing.xxxSmall_4,
     },
     centered: {
-        verticalAlign: "middle",
+        // Manually align the bottom of start/end icons with the text baseline.
+        verticalAlign: "-10%",
```

Comment:
**question:** have you tested this change in different browsers? I'm not fully sure if this will be consistent or not. Also, what would happen if the button text has a different font size?

Reply:
if the button text has a different font size, the icon will sit closer to the baseline of the text instead of upper-middle area. the main change here has to do with removing this style from the actual link text, which was what caused it to sit lower than expected.
<img width="255" alt="image" src="https://github.com/Khan/wonder-blocks/assets/77523470/eacd0499-2f9d-44eb-baac-a5c2742aa260">

Reply:
Got it, thanks for the example. Let's land this in `webapp` and see how it works with all the different instances.